packages/lodestar-cli/src/cmds/account/cmds/validator/import.ts

```diff
@@ -35,7 +35,7 @@ Ethereum Foundation utility.",
 
   examples: [
     {
-      command: "account validator import --testnet medalla --directory $HOME/eth2.0-deposit-cli/validator_keys",
+      command: "account validator import --network medalla --directory $HOME/eth2.0-deposit-cli/validator_keys",
```

Comment:
We should replace all mentions of medalla (and any other dead testnets) with pyrmont (in a future issue / PR)

Reply:
made an issue for it #1972